bot/seasons/evergreen/snakes/snakes_cog.py

```diff
@@ -950,8 +950,8 @@ def predicate(reaction_: Reaction, user_: Member) -> bool:
                 "48 hours. The staff then selected the best features from all the best teams, and made modifications "
                 "to ensure they would all work together before integrating them into the community bot.\n\n"
                 "It was a tight race, but in the end, <@!104749643715387392> and <@!303940835005825024> "
-                "walked away as grand champions. Make sure you check out `!snakes sal`, `!snakes draw` "
-                "and `!snakes hatch` to see what they came up with."
+                "walked away as grand champions. Make sure you check out `.snakes sal`, `.snakes draw` "
+                "and `.snakes hatch` to see what they came up with."
```

Comment:
Hey, thanks for your contribution. Although, the prefix is dynamic and set by the constant `bot.constants.Client.prefix`, you should use it instead of hardcoding it.